pints/_optimisers/__init__.py

```diff
@@ -455,6 +455,10 @@ def __init__(
         self._unchanged_threshold = 1          # smallest significant f change
         self.set_max_unchanged_iterations()
 
+        # Maximum evaluations
+        self._max_evaluations = None
+        self.set_max_evaluations()
```

Comment:
Can remove line 459? As far as I can see, line 460 sets already the number of max evaluations to None.